### DISCUSSION TITLE
Are you ready for exchange? It's coming!

### DISCUSSION THREAD

**Discussion by Rikkyf** [CyberChainXyz/go-cyberchain#6] [General]
![1](https://github.com/user-attachments/assets/0963b9e5-f296-4acb-8b2c-53b3bf7f27ad)

**Comment by dmitrysv85** [comment]
I think it would be better to start with xeggex or nokyc ...

**Reply by minertes** [reply]
https://app.exbitron.com/exchange/?market=CCX-USDT  first exchange imo

**Reply by qusai1991234** [reply]
this exchange do not have volume so it is like we are not listing yet

**Reply by minertes** [reply]
traders make the volume, not exchanges:)

**Reply by qusai1991234** [reply]
bro in general this exchange do not have volume cuz no one use it

**Reply by minertes** [reply]
i see. Yes it is not an xeggex or ogre but a safe place to trade New coins. Better than otc imo. i am using for pre-xeggex trades generally:)

**Reply by qusai1991234** [reply]
btw why we do not have telegram or discord

**Reply by minertes** [reply]
Thats weird. Hopefully they open one soon.

**Reply by StianNOR** [reply]
> btw why we do not have telegram or discord

https://discord.gg/2K99ewZdjr

unofficial one is online.

**Reply by BelzebVct** [reply]
yesterday after downloading the miner and trying it, I found my wallet empty! congratulations to the creator! stay away from these shitty human beings.

**Reply by nathanccxv** [reply]
> yesterday after downloading the miner and trying it, I found my wallet empty! congratulations to the creator! stay away from these shitty human beings.

I'm sorry to hear you're facing issues, but Transparency and trust are the core of what we do. Here's more info about xMiner:

https://github.com/orgs/CyberChainXyz/discussions/9

**Comment by crazy32000ka** [comment]
大哥们有没有linux的教程啊

**Reply by nathanccxv** [reply]
The steps are the same as on Windows: download the corresponding release file and run the respective .sh file. What issue are you encountering?
